modules/connatixBidAdapter.js

```diff
@@ -41,10 +43,28 @@ export function getBidFloor(bid) {
   }
 }
 
+export function validateBanner(mediaTypes) {
+  if (!mediaTypes[BANNER]) {
+    return true;
+  }
+
+  const banner = deepAccess(mediaTypes, BANNER, {});
+  return (Boolean(banner.sizes) && isArray(mediaTypes[BANNER].sizes) && mediaTypes[BANNER].sizes.length > 0);
+}
+
+export function validateVideo(mediaTypes) {
+  if (!mediaTypes[VIDEO]) {
+    return true;
+  }
+
+  const video = deepAccess(mediaTypes, VIDEO, {});
```

Comment:
redundant deepAccess here

you already validated that mediaType[VIDEO] exists

Reply:
ok so you can update the const video to not use deepAccess at least :) 

const video = mediaTypes.VIDEO that is already check above or better add a const for mediaTypes[VIDEO] and use it in the above if and here